### PR TITLE
Update new mercure hub url in test

### PR DESCRIPTION
**Pull request by castroCrea**

### CODE REVIEW

File: tests/Debug/TraceablePublisherTest.php [hunk 1]
```diff
@@ -24,8 +24,8 @@
  */
 final class TraceablePublisherTest extends TestCase
 {
-    const URL = 'https://demo.mercure.rocks/hub';
-    const JWT = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJtZXJjdXJlIjp7InN1YnNjcmliZSI6WyJmb28iLCJiYXIiXSwicHVibGlzaCI6WyJmb28iXX19.LRLvirgONK13JgacQ_VbcjySbVhkSmHy3IznH3tA9PM';
+    const URL = 'https://demo.mercure.rocks/.well-known/mercure';
+    const JWT = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJtZXJjdXJlIjp7InN1YnNjcmliZSI6WyJmb28iLCJiYXIiXSwicHVibGlzaCI6WyJmb28iXX19.afLx2f2ut3YgNVFStCx95Zm_UND1mZJ69OenXaDuZL8';
```
Comment (dunglas) [inline tests/Debug/TraceablePublisherTest.php:28]: Why this change?

Comment (dunglas) [reply tests/Debug/TraceablePublisherTest.php:28]: Ok got it, because the signature changed.

**Comment by dunglas** [comment]
Thanks @castroCrea!

**Comment by castroCrea** [comment]
Thanks @dunglas,
Can you do a release ?
